### ISSUE TITLE
Email unsubscribe and preference update

### ISSUE THREAD

**Issue by d33pspace** [d33pspace/RenewalWebsite#114]
The following links are used in our communications for preference update and unsubscribe. Please give them a good testing. It's not necessary to test the emails themselves, you can just modify and use these links directly. It's also not necessary to make sure the changes actually took place after submission. I'll handle that. Just test the website functionality and interface in English and Chinese. (I know for one thing we need to add the "please wait" spinner to the unsubscribe button.) See what happens with empty fields, invalid email addresses, etc.

Email preference update:
https://staging.renewal365.org/preference?email=jimmy@renewal365.org&salutation=Jimmy

Unsubscribe:
https://staging.renewal365.org/preference?email=jimmy@renewal365.org

**Comment by d33pspace** [comment]
I see you've already tested the unsubscribe page and I opened another ticket for the please wait spinner. But I'll leave this ticket open for now as a reminder to please take a good look at both of these pages.

**Comment by elainetao829** [comment]
Hi @d33pspace,

Thanks for reminding me test this functionality. I think we could have some update or discussion at least over the following issues,
1. The preference page is a link from the email. And there is the email address in the title. Why the system provides a free text field for user to enter any email address?
I think we do not need this field. Please think this over.  
![image](https://user-images.githubusercontent.com/42002784/45597437-09fa5580-b9ff-11e8-8673-09ac78be2d94.png)

2. There is no check for both email and first name field, i.e. no matter user leaves them empty, or enter some invalid value, the system would not pop out any error or warning. 
This is a bug and we should add some messsages as the subscription page, which are,
For English,
Enter your name 
Enter valid email address. 
For Chinese,
请输入你的名字。
输入有效的电邮地址。 

3. The Chinese on this page means "setting" （设置） instead of preference （偏好）. So the translation can be updated into,
Tab： 设置-->偏好
改变日新中心简讯的设置…… --> 改变日新中心通讯的偏好……
![image](https://user-images.githubusercontent.com/42002784/45597545-95c0b180-ba00-11e8-8e30-252499422394.png)

**Comment by d33pspace** [comment]
Taking these in reverse order, (3) I've made the change, issue (2) I've created #120, and (3) this field is for the user to enter a new email address which changes the email address we use to send them Renewal communications. This is a fairly common request. For clarity, I've changed the English label to "Change communications email" and the Chinese to "改变简讯电子邮件" Does this make sense now and is the translation acceptable?
